micrometer-tracing-tests/micrometer-tracing-test/src/main/java/io/micrometer/tracing/test/simple/TracingAssertions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.tracing.test.simple;
+
+import java.util.Collection;
+
+import io.micrometer.tracing.exporter.FinishedSpan;
+
+/**
+ * Entry point assertions.
+ *
+ * @author Tadaya Tsuyukubo
+ * @since 1.0.0
+ */
+public class TracingAssertions {
```

Comment:
I think this should extend the AssertJ assertion class. That way you can have a single class that has all assertions.

Reply:
Unfortunately AssertJ `Assertions` conflicts with `assertThat(Collection<FinishedSpan> actual)` for its collection assertion. So, we cannot inherit assertj core assertions.

Reply:
I was afraid you would say it ☹️

Reply:
yeah, not inheriting is also [AssertJ recommendation](https://joel-costigliola.github.io/assertj/assertj-core-custom-assertions.html#single-assertion-entry-point) for this case.
> The solution is simple, don't make your classes inherit from org.assertj.core.api.Assertions and use import static org.assertj.core.api.Assertions.assertThat to access core assertions.

At least, this class gives a single static import for those three custom assertion classes rather than individually static importing them.

Reply:
Hmm I'm pretty sure the recommendation a couple of years ago was the opposite but maybe I'm wrong